controllers/servers/csi/decorators.py

```diff
@@ -50,3 +50,14 @@ def _set_sync_lock(lock_id, lock_request_attribute, error_response_type,
         return handle_exception(ex, context, grpc.StatusCode.ABORTED, error_response_type)
     logger.info("finished {}".format(controller_method_name))
     return response
+
+
+def register_csi_plugin(registration_map):

```

Comment:
why not just import SVC_REGISTRATION_MAP?

Reply:
wanted that this decorator could serve non SVC functions in the future

Reply:
I don't believe that we would have something similar in other storages, and if we do, we could use the same map - it's keyed on endpoints..

Reply:
right, changed it